docs/testcases/UI/Run_tools_in_sandbox/Launch_parameters/2642/2642_6.md

```diff
@@ -0,0 +1,29 @@
+# Check running instances restriction for launch tool with configured Cluster
+
+Test verifies that the restriction of the running instances count works correctly for case of Auto-Scaled Cluster runs
+
+**Prerequisites**:
+- Admin user
+- Non-admin user
+- Perform [2642_5](2642_5.md) case
+
+| Steps | Actions | Expected results |
+| :---: | --- | --- |
+|   1   | Login as the admin user from the prerequisites | |
+|   2   | Open the **Tools** page | |
+|   3   | Select the test tool | |
+|   4   | Open **Settings** tab | |
+|   5   | Expand the **Exec environment** section | |
+|   6   | Click the **Configure cluster** control | |
+|   7   | Click the **Cluster** tab | |
+|   8   | Set value `2` into ***Child nodes:*** field | |
+|   9   | Click the **OK** button | |
+|  10   | Click **Save** button | |
+|  11   | Login as the non-admin user from the prerequisites | |
+|  12   | Open the **Tools** page | |
+|  13   | Select the test tool | |
+|  14   | Launch a tool with default settings | **Launch** pop-up that contains error message `You have exceeded maximum number of running jobs (3).` appears |
+|  15   | Click **Launch** button | Error message `Launch of new jobs is restricted as [<non-admin user>] user will exceed [<user-contextual-limit>] runs limit [3]` appears, where <br> `<non-admin user>` is the non-admin user from the prerequisites, <br> `3` is the value specified at step 6 of the Preparations |
```

Comment:
Add:
`... at step 6 of the Preparations of the [2642_5](2642_5.md) case`

Reply:
Done